frontend/src/components/shared/Radio/Radio.tsx

```diff
@@ -83,13 +83,21 @@ class Radio extends React.PureComponent<Props, State> {
                     marginBottom: 0,
                     marginTop: 0,
                     paddingRight: fontSizes.twoThirdSmDefault,
-                    backgroundColor: $isFocused ? colors.secondaryBg : "",
+                    backgroundColor: $isFocused ? colors.darkenedBgMix15 : "",
                     borderTopLeftRadius: radii.md,
                     borderTopRightRadius: radii.md,
                     borderBottomLeftRadius: radii.md,
                     borderBottomRightRadius: radii.md,
                   }),
                 },
+                RadioMarkOuter: {
+                  style: ({ $checked }: { $checked: boolean }) => ({
```

Comment:
Nit: There should exist a type for Radio with `{ $checked: boolean }` in Base. Try using that instead.